<launch>
	<node pkg="dji_sdk_control" type="pid_ctrl_node" name="nodeOne" output = "screen">
	<!-- node parameters -->
	<param name="Kp_pos" value="0.2" type="double" />
	<param name="Ki_pos" value="0.0" type="double" />
	<param name="Kd_pos" value="0.0" type="double" />
	</node>
</launch>
